### PR TITLE
Fixing bug with inserting image and couldn't type afterward

### PR DESCRIPTION
**Pull request by ivanaveliskova**
Currently in PR app: https://monosnap.com/file/MrOZzCa9ccfGtExowXyPYXUksuxkJr

With this update: https://monosnap.com/file/cdfcbRD3yqWOUz9UvW1SWHorjhAZSb

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -104,7 +104,6 @@ var defaults = {
 
   function onChange(change) {
     if (!canBeEmpty) return null;
-    var hasSave = change.flags.hasOwnProperty('save');
```
Comment (ivanaveliskova) [inline dist/index.js:107]: This wasn't being used anywhere so clean up

File: lib/index.js [hunk 2]
```diff
@@ -73,7 +73,6 @@ function StickyInlines(opts) {
 
   function onChange(change) {
     if (!canBeEmpty) return null
-    const hasSave = change.flags.hasOwnProperty('save')
```
Comment (ivanaveliskova) [inline lib/index.js:76]: this wasn't being used anywhere so it's cleanup

File: lib/index.js [hunk 3]
```diff
@@ -99,16 +98,17 @@ function StickyInlines(opts) {
    */
 
   function onSelect(event, change) {
-    const selection = findRange(window.getSelection(), change.value);
-    const selectionIsAtEndOfInline = change.value.focusInline &&
-      selection.focusOffset === change.value.focusInline.text.length;
+    if(!change.value.focusInline) return null
```
Comment (ivanaveliskova) [inline lib/index.js:101]: With some modifications to anything that wasn't available, most of this is actually from: https://github.com/aha-app/slate-sticky-inlines/blob/slate-0.37/lib/index.js

**Review by Jacfem — COMMENTED** [review]
Hmm, I'll dig into this more soon, but should clicking next to/below the image be working with this change as well? I'm actually not seeing anything changed when I try to click to the left of the image or below the image which I thought was the major breaking bug -- I can only get out of the 'focused' state if I use the arrow keys 🤔

**Comment by ivanaveliskova** [comment]
If you click next to the image, you should be able to type properly. Can you share a video explaining what you might be doing?

**Comment by Jacfem** [comment]
Ok cool, I wasn't seeing that behavior locally, I'll re-install etc and keep you posted!

**Comment by ivanaveliskova** [comment]
Did you link it to guru-ui? Because that's where we pull it into the plugins. Not in slate-tools

**Comment by Jacfem** [comment]
It's highly possible I don't have it linked correctly! Linking for sure now 👀

**Comment by Jacfem** [comment]
That must've done it, I see it working now @ivanaveliskova! Awesome job. I'll read the actual code changes now :)

**Review by Jacfem — APPROVED** [review]

**Review by draffauf — APPROVED** [review]

**Pull request merged**